CONTRIBUTING.md

```diff
@@ -37,10 +37,11 @@ able to use it. From the top level of the repo, there are three commands availab
   dependencies (`utils`, `core`, `browser`, etc), and all packages which depend on it (currently `gatsby` and `nextjs`))
 - `yarn build:dev:watch`, which runs `yarn build:dev` in watch mode (recommended)
 
-You can also run a production build via `yarn build`, which will build everything except for the tarballs for publishing to NPM.
-You can use this if you want to bundle Sentry yourself. The build output can be found in the packages `build/` folder, e.g. `packages/browser/build`.
-Bundled files can be found in `packages/browser/build/bundles`.
-Note that there are no guarantees about the produced file names etc., so make sure to double check which files are generated after upgrading.
+You can also run a production build via `yarn build`, which will build everything except for the tarballs for publishing
```

Comment:
Not sure if this change was intentional but it looks like we had a lint issue in `Contributing.md` which I fixed in #10385. looks like this reverts the fix 🤔

Reply:
hmm weird this was done by prettier I think 🤔 I'll rebase on develop again!